src/main/java/org/fundacion/jala/converter/core/facade/ExtractFacade.java

```diff
@@ -67,7 +67,7 @@ public static String getMetadataExtract(final MultipartFile file, final Boolean
         String pathFile = null;
```

Comment:
The method signature should reach 120 characters

Reply:
Solved